### DISCUSSION TITLE
base 怎么进行多数据集多轮训练

### DISCUSSION THREAD

**Discussion by codingforwhat** [zai-org/ChatGLM3#1244] [Q&A] [answered]
请问有没有从已有微调模型继续训练的方法？现在这个微调的数据格式可以用于微调base模型吗？之前的微调代码怎么修改epoch进行多轮训练

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
可以适配 base，但是base不是对话模型
你把max step 改成epoch就行

**Reply by codingforwhat** [reply]
MAX_SOURCE_LEN=512
MAX_TARGET_LEN=128
DEV_BATCH_SIZE=1
GRAD_ACCUMULARION_STEPS=1
MAX_STEP=5000
SAVE_INTERVAL=50
MAX_SEQ_LEN=512
修改这里max_steps后面的值是吗？也就是max_steps就是epoch
